tap_salesforce/client.py

```diff
@@ -137,6 +137,7 @@ def get_tables(self) -> Generator[Tuple[str, Dict[str, Field], str], None, None]
             ("Invoice__c", "LastModifiedDate"),
             ("Trial__c", "LastModifiedDate"),
             ("Task", "LastModifiedDate"),
+            ("Event", "LastModifiedDate"),
```

Comment:
This is so obviously wrong!

Reply:
panicked for a sec by the mix signals you are sending